Lab1/Lab1/Models/StudentInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lab1.Models
+{
+    [DataContract]
```

Comment:
It's required attribute?

Reply:
Yes, it's required attribute. This attribute means that your class can be serialized by default Json serializer. In this class all field and properties with attribute [DataMember] will serialized as data. But i agree, that i can find better way of realization of this.